src/FreeAge/client/render_window.cpp

```diff
@@ -685,8 +685,8 @@ void RenderWindow::GrabMouse() {
         result = ::XGrabPointer(
             QX11Info::display(),
             winId(),
-            False,
-            ButtonPressMask | ButtonReleaseMask | PointerMotionMask | FocusChangeMask | EnterWindowMask | LeaveWindowMask,
+            True,
+            0,
```

Comment:
On my linux, I cannot wheel scroll if the cursor is grabbed. I couldn't find the mask for the wheel, so I did this and its working...